src/custom/components/Settings/SettingsMod.tsx

```diff
@@ -127,14 +128,36 @@ export default function SettingsTab({ className, placeholderSlippage, SettingsBu
 
   const theme = useContext(ThemeContext)
 
-  const [expertMode, toggleExpertMode] = useExpertModeManager()
+  const [expertMode, toggleExpertModeAux] = useExpertModeManager()
+  const toggleExpertMode = useCallback(() => {
+    ReactGA.event({
```

Comment:
It's a toggle function, so this event would always fire a "confirmation" signal in GA even if the user is disabling expert mode

Reply:
Yes, this is same as @alfetopito mentioned. I will make sure the event is slightly different if we are disabling